src/Tizen.NUI.Components/Controls/Navigation/AppBar.cs

```diff
@@ -182,9 +184,45 @@ public View NavigationContent
             }
         }
 
+        /// <summary>
+        /// Title text of AppBar.
+        /// SetTitle sets title text to the default title content.
+        /// If TitleContent is not TextLabel, then Title does not set title text of the TitleContent.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Title
```

Comment:
I think no need this property, 

Because, TitleContent was provide by API user, So, he can control value of Text by himself. 
It just make a confusing

``` c#
 appbar.Title = "aaa";
 assert(appbar.Title == "aaa"); // it will be fail if TitleContent is not TextLabel
```
``` c#
 appbar.TitleContent = new TextLabel();
 appbar.Title = "aaa";
 assert(appbar.Title == "aaa"); // it is pass

 appbar.TitleContent = new TextLabel();
 assert(appbar.Title == "aaa"); // it is fail
```

Reply:
If a you want to give a responsibility of set a title on AppBar, 
You need to hold a title value, and you should update when TitleContent was updated
like below
``` c#
  public string Title {get; set;}
 
  public View TitleContent
  {
       set 
       {
             if (value is TextLabel label) 
             {
                   label.Text = Title;
             }
       }
  }

```

Reply:
I want to request to you that separate between data and UI, in this case, Title is data, View is UI, 

UI could be change by want, but Data(title) should not be reflected by those changes.
And
Title also could be changed by want, but UI no need to change

Reply:
Thank you for your comments.
I understand your point and I would like to explain why I added "Title" property.

As you said, user can set TextLabel to "TitleContent".
Since NUI supports theme style by its type (e.g. TextLabelStyle is defined for default TitleContent), if "TitleContent" is not TextLabel, then we cannot apply Tizen UX for AppBar Title to the "TitleContent".

In other words, if user sets any Control to "TitleContent" except TitleLabel, then Tizen UX for AppBar Title is not applied.

So I wanted to provide "Title" for most of users who just wants to set text on Title area.
If user wants to set something else, then they can use "TitleContent".

Reply:
I considered your implementation before, but I had a problem as the following case.

```
appbar.TitleContent = new TextLabel();
appbar.Title = "aaa";
(appbar.TitleContent as TextLabel).Text = "bbb";
```

In the above case, the data "Title" would be still "aaa", then it is not synchronized with the UI text "TitleContent.Text".

I thought that it is not that good so I chose the current implementation.
I would like to change the current implementation to your suggestion if the above case would be fine from application developer's perspective.

Reply:
I think, it is not problem, 
Because It is **intent** of user,

This rule was also applied on data binding

If user set a value on property that was binding some value, 
how to work? 
in this case binding was broken because it is intent of user. 

I think it is basically same with our case

What is your opinion on this case?
1. `appbar.Title` should be "bbb"?
2. or `(appbar.TitleContent as TextLabel).Text` should not change to "bbb" so still it's value was "aaa"?

if your think is 1, 
We can track changes of Text on TextLabel by using `PropertyChanged` event and update a value of `Title`
But, I don't think it is need

Reply:
Thank you very much for your answer!
Your comment is so clear that I think I can apply the way of thinking to other cases as well.

About the preference between 1 and 2, I agree with you that I don't think 1 is needed.
And 2 is not what I want.

So I am going to apply your suggestion about implementing "Title" and "TitleContent".
Thank you :)